src/Components/Scribe/formDetails.ts

```diff
@@ -97,63 +122,76 @@ export const DAILY_ROUND_FORM_SCRIBE_DATA: Field[] = [
     example: "General Ward",
     description:
       "A string to store the department or ward where the patient is admitted.",
+    validator: (value) => typeof value === "string",
   },
   {
     friendlyName: "bp",
     id: "bp",
-    default: "{ systolic: undefined, diastolic: undefined, mean: undefined }",
+    default: { systolic: null, diastolic: null, mean: null },
     type: "{ systolic: number, diastolic: number, mean: number }",
     example: "{ systolic: 120, diastolic: 80, mean: 100 }",
     description:
       "An object to store the blood pressure of the patient. It contains two integers, systolic and diastolic. Output mean is calculated from these two.",
+    validator: (value) => {
+      if (typeof value !== "object") return false;
+      if (value.systolic && typeof value.systolic !== "number") return false;
+      if (value.diastolic && typeof value.diastolic !== "number") return false;
+      if (value.mean && typeof value.mean !== "number") return false;
+      return true;
+    },
```

Comment:
@shivankacker There is already a re-usable validator for blood pressure defined. How about using that instead of manually redefining validations?

<img width="966" alt="image" src="https://github.com/user-attachments/assets/a48fc9e7-4a41-4e33-9104-eaa208188577">

Reply:
It doesn't check for types thought, scribe might return string instead of number in very rare halucinations. Also because we are in edit mode only, there is no need to validate null checks. Edge cases like user might be unsure of systolic and would want to input that after dictation.